# Key Method used to detect conserved ORFs
For detecting conserved ORFs methods used integrated computational pipeline such as Comparitive genomics, sequence alignment tools, Ribosome profiling. It uses dual approach both computational as well as experimental validation, ensures that identified ORFs were both conserved and functional.
Comparitive genomics:- This method used to compare genes across multiple species(human, mouse, nematode, fruitfly, zebrafish)
Sequence alignment tools:- To identify conserved short ORFs
Ribosome profiling (Ribo-seq):- To confirm active translation of candidate ORFs.

# One interesting Result
One interesting result discovered that hundreds of previously unannotated conserved sORFs encoding functional micropeptides. Some of these micropeptides were shown to play essential development roles, challenging the assumptions that non-coding regions lack protein-coding potential.


# How conservation was quantified
Conservation was quantified using various methods, first one is :phylogenetic conservation scores such as PhastCons, phyloP across species. Codon-level alignment metrics to detect evolutionary constraints. Comparative sequence identity across orthologous ORFs.Then ORFs with high conservation scores were prioritized as candidated for functional importance.

# Connection to my ORF comparision
It connects with my ORF comparision by performing functions and gives the output which it gives using that methods. It shows higest score of ORFs and consider it as similar to those methods.
I write lines of codes in my ORF comparision in which it took two different sequences of RNA and then find ORF separately, and giving output comparing both sequences and their protein it encodes, Length of ORF, and length of protein.

